package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "chalk-template",
 	"version": "0.0.0",
-	"description": "TODO",
+	"description": "Use tagged template literals with Chalk",
```

Comment:
That's not really the purpose though. The purpose is to apply colors using templating syntax.